nodl_python/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>nodl_python</name>
+  <version>0.0.1</version>
+  <description>CLI and parsing utilities for the ROS 2 NoDL</description>
```

Comment:
CLI and parsing are going to be in the same package?

Reply:
For now I think that makes sense. The CLI may not prove useful in the long run once the lib is stable.